app/components/card-breakdown-chart.js

```diff
@@ -0,0 +1,56 @@
+import Ember from 'ember';
+import layout from '../templates/components/card-breakdown-chart';
+
+export default Ember.Component.extend({
+  layout: layout,
+
+  bars: function () {
+    var deck = this.get('deck');
+    var cardGroups = deck.get('cardGroups');
+    var mainCount = deck.get('mainCount');
+    var stats = {
+      Creature: {
+        count: 0,
+        width: 0
+      },
+      Artifact: {
+        count: 0,
+        width: 0
+      },
+      Sorcery: {
+        count: 0,
+        width: 0
+      },
+      Instant: {
+        count: 0,
+        width: 0
+      },
+      Land: {
+        count: 0,
+        width: 0
+      },
+      Enchantment: {
+        count: 0,
+        width: 0
+      }
+    };
+
+    cardGroups
+      .filterBy('board', 'main')
+      .forEach((group) => {
+        var stat = stats[group.get('card.mainType')];
+        var count = group.get('count');
+        stat.count += count;
```

Comment:
I think this might bomb if the hard-coded types don't encompass what the deck contains.